src/test/java/org/jabref/logic/journals/AbbreviationTest.java

```diff
@@ -101,4 +103,17 @@ void testDefaultAndShortestUniqueAbbreviationsAreSame() {
         Abbreviation abbreviation = new Abbreviation("Long Name", "L N");
         assertEquals(abbreviation.getAbbreviation(), abbreviation.getShortestUniqueAbbreviation());
     }
+
+    @Test
+    void testToString() {
+      Abbreviation abbreviation = new Abbreviation("Long Name", "L N", "LN");
+      assertEquals(abbreviation.toString(), "Abbreviation{name=Long Name, abbreviation=L N, medlineAbbreviation=L N, shortestUniqueAbbreviation=LN}");
+    }
+
+    @Test
+    void testEquals() {
+      Abbreviation abbreviation = new Abbreviation("Long Name", "L N", "LN");
+      assertTrue(abbreviation.equals(abbreviation));
```

Comment:
I detect that this code is problematic. According to the [Correctness (CORRECTNESS)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#correctness-correctness), [SA: Self comparison of value with itself (SA_LOCAL_SELF_COMPARISON)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#sa-self-comparison-of-value-with-itself-sa-local-self-comparison).
This method compares a local variable with itself, and may indicate a typo or a logic error. Make sure that you are comparing the right things.

Reply:
Thanks for the feedback! This test originally was meant to cover the branch in the implementation of the equals method, where it returns true for being compared to itself. But seeing your linked references, I agree that it makes more sense to leave it out - I have adjusted the test now.

Reply:
Happy to help :)